wolfcrypt/benchmark/benchmark.c

```diff
@@ -4496,7 +4509,21 @@ static void bench_rsa_helper(int doAsync, RsaKey rsaKey[BENCH_MAX_PENDING],
     #if !defined(WOLFSSL_RSA_VERIFY_INLINE) && \
                     !defined(WOLFSSL_RSA_PUBLIC_ONLY)
         DECLARE_ARRAY_DYNAMIC_EXE(out, byte, BENCH_MAX_PENDING, rsaKeySz, HEAP_HINT);
+        if (out[0] == NULL) {
+            printf("malloc failed\n");
+            goto exit;
+        }
     #endif
+    if (enc[0] == NULL) {
+        printf("malloc failed\n");
```

Comment:
remove debug statements

Reply:
Whoops -- replaced them with `ret = MEMORY_E`.  Thanks.

Note that `bench_aesgcm_internal()` and `bench_aesccm()` still have `printf("malloc failed\n")` in them -- more extensive refactors are needed for proper error reporting in these.  Let me know if you think it's worth doing now, but there are many other existing examples of this code pattern in benchmark.c, that are similarly crying out for a refactor that's probably not worth the effort.